### PR TITLE
Enable CodeQL analysis on sources

### PR DESCRIPTION
**Pull request by asomsiko**
CodeQL is a semantic code analysis engine.
It searches for security issues and allows custom
queries.

### CODE REVIEW

File: .github/codeql/codeql-config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: "CodeQL python config"
+
+paths:
+  - "model-optimizer"
```
Comment (ilya-lavrenov) [inline .github/codeql/codeql-config.yml:4]:
```suggestion
  - "tools/mo"
```

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with: 
+        submodules: recursive
+
+    - name: Install dependencies
+      run: |
+        sudo apt --assume-yes install python3-setuptools
+        python3 -m pip install --upgrade pip
+        python3 -m pip install -r ./model-optimizer/requirements.txt
```
Comment (ilya-lavrenov) [inline .github/workflows/codeql-analysis.yml:27]:
```suggestion
        python3 -m pip install -r tools/mo/requirements.txt
```

File: .github/workflows/codeql-analysis.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '27 4 * * 2'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-18.04
```
Comment (ilya-lavrenov) [inline .github/workflows/codeql-analysis.yml:15]:
```suggestion
    runs-on: ubuntu-latest
```

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**